analysis/deep2_r23_o32/plotting/manual_bin_grid.py

```diff
@@ -23,7 +23,7 @@ def graph_bins(fitspath, n_split):
     """
 
     bin_info = join(fitspath, filename_dict['bin_info'])
-    individual_info = join(fitspath, filename_dict['individual_prop'])
+    individual_info = join(fitspath, filename_dict['indv_prop'])
```

Comment:
This is also being addressed [here](https://github.com/astrochun/Zcalbase_gal/pull/84/files#diff-6192c50fa16612dc9f253e5f71bc7f8bfd06f05de3268b910cf166374100b9dc) in #84